src/test/scala/com/exasol/cloudetl/kafka/KafkaConsumerPropertiesTest.scala

```diff
@@ -373,8 +398,8 @@ class KafkaConsumerPropertiesTest extends AnyFunSuite with BeforeAndAfterEach wi
     val metadata = mock[ExaMetadata]
     val connectionInformation = mock[ExaConnectionInformation]
     when(metadata.getConnection("MY_CONNECTION")).thenReturn(connectionInformation)
-    when(connectionInformation.getUser()).thenReturn("")
-    when(connectionInformation.getPassword())
+    when(connectionInformation.getUser).thenReturn("")
```

Comment:
Please revert here and other places. 

I think you had formatting rule to remove them.